docs/03-extensions/stripe/reference/environment-variables.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: "Environment variables"
+description:
+  "This reference lists environment variables used by the Stripe extension"
+sidebar_position: 1
+---
+
+<p>{frontMatter.description}</p>
+
+:::info
+
+Your API keys can be found on your
+[Stripe Dashboard](https://dashboard.stripe.com/account/apikeys)
+
+:::
+
+| Name                                    | Description                                                                                                                                     | Default value |
+| --------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| FRONT_COMMERCE_STRIPE_SECRET_KEY        | **Required** Your Stripe Secret key                                                                                                             | `undefined`   |
+| FRONT_COMMERCE_STRIPE_PUBLISHABLE_KEY   | **Required** Your Stripe Secret key                                                                                                             | `undefined`   |
```

Comment:
**thought:** if a variable is required, there is no default value.

**suggestion:** remove them.

```suggestion
| FRONT_COMMERCE_STRIPE_SECRET_KEY        | **Required** Your Stripe Secret key                                                                                                             |    |
| FRONT_COMMERCE_STRIPE_PUBLISHABLE_KEY   | **Required** Your Stripe Secret key                                                                                                             |    |
```

Reply:
removed